README.md

```diff
@@ -74,6 +74,14 @@ The username is `admin`, and the password can be retrieved by running:
 oc get dc sso -n integreatly-rhsso -o jsonpath='{.spec.template.spec.containers[0].env[?(@.name=="SSO_ADMIN_PASSWORD")].value}'
 ```
 
+## Setting up your cluster to be OSD like
+
+To setup your cluster to have dedicated admins run the ```dedicated-setup.sh``` script
+```
+cd scripts
```

Comment:
Make command would be nice and save using changing directory

Reply:
You can run `./scripts/dedicated-setup.sh` to save a `cd`

Reply:
More the make command as per how we do other things in the repo. Not a huge issue

Reply:
your wish is my command :)